app/react/Library/actions/exportActions.js

```diff
@@ -0,0 +1,76 @@
+import superagent from 'superagent';
+import { toUrlParams } from 'shared/JSONRequest';
+import { actions } from 'app/BasicReducer';
+import { notify } from 'app/Notifications/actions/notificationsActions';
+import { t } from 'app/I18N';
+import { processFilters } from './libraryActions';
+
+export function triggerLocalDownload(content, fileName) {
+  const url = window.URL.createObjectURL(new Blob([content]));
+  const link = document.createElement('a');
+  link.href = url;
+  link.setAttribute('download', fileName);
+  document.body.appendChild(link);
+  link.click();
+  link.parentNode.removeChild(link);
+}
+
+export function exportEnd() {
+  return (dispatch, getState) => {
+    triggerLocalDownload(
+      getState().entityExport.exportContent,
+      getState().entityExport.exportFileName
+    );
+
+    dispatch(actions.set('exportProcessing', false));
+    dispatch(actions.set('exportContent', ''));
+    dispatch(actions.set('exportFileName', ''));
+  };
+}
+
+export function extractFileName(contentDisposition) {
+  const startIndex = contentDisposition.indexOf('filename="') + 10;
+  const endIndex = contentDisposition.length - 1;
+  return contentDisposition.substring(startIndex, endIndex);
+}
+
+export function exportDocuments(storeKey) {
+  // eslint-disable-next-line max-statements
```

Comment:
Please refactor this to contain less statements instead of disabling the lint rule.